styles.css

```diff
@@ -369,3 +368,97 @@
   padding-top: 20px;
   list-style-type: none;
 }
+
+.contact-me {
+  display: flex;
+  flex-direction: column;
+  border-top-left-radius: 65px;
+  height: 779px;
+  background: #6070ff;
+}
+
+footer .content h2 {
+  font-family: 'poppins', sans-serif !important;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  color: #fff;
+  padding-left: 20px;
+}
+
+footer .content p {
+  font-family: 'poppins', sans-serif !important;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #ebebff;
+}
+
+form .name {
+  width: 327px;
+  height: 48px;
+  background: #fff;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  margin: 48px 0 20px 0;
+}
+
+form .email {
+  width: 327px;
+  height: 48px;
+  background: #fff;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  margin: 0 0 18px 0;
+}
+
+form textarea {
+  background: #fff;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  width: 327px;
+  height: 170px;
+}
+
+.btn {
+  width: 108px;
+  height: 24px;
+  font-family: 'poppins', sans-serif !important;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  background-color: #fff;
+  border: none;
+}
+
+form .btn {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px 16px;
+  width: 140px;
+  height: 48px;
+  background: #fff;
+  border-radius: 8px;
+  margin: 22px 211px 0 0;
+}
+
+.btn:hover {
+  background-color: #6070ff !important;
+  transition: all 0.5s ease-in-out;
+  color: #fff !important;
```

Comment:
- [ ] Kindly avoid overusing `important` in your project, it is not a best practice 👍🏿